### PR TITLE
PD-173: RAID level configuration

### PR DESCRIPTION
**Pull request by andreipope**
@harsh-px @nathan-px Hey guys, please review this when you get the chance. Thanks!

P.S.: We can also tell the users they can append "&misc=-raid%2010" or "&misc=-raid%200" to the URL, but I feel that's a bit more fiddly.

### CODE REVIEW

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/advanced-installation-parameters.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+1. Use the steps from the [Generate the specs](/portworx-install-with-kubernetes/on-premise/other/#generate-the-specs) section to generate a Kubernetes manifest.
+
+2. Download the manifest and use the `args` section of the Portworx `DaemonSet` to specify the RAID level.
+
+  * Below is an example for specifying RAID 0:
+
+    ```text
+    args:
+      ["-k", "etcd:http://70.0.0.65:2379", "-c", "kockica-11", "-d", "eth1", "-m", "eth1", "-s", "/dev/sdb", "-secret_type", "k8s", "-raid", "0",
```
Comment (harsh-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/advanced-installation-parameters.md]: add an explicit note that rest of the arguments apart from raid here are just examples and users should not use them.

Comment (andreipope) [reply content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/advanced-installation-parameters.md]: > add an explicit note that rest of the arguments apart from raid here are just examples and users should not use them.

I added a note.

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/advanced-installation-parameters.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Portworx supports two RAID level configurations for local storage: RAID 0, and RAID 10.
+
+Follow these steps to specify the RAID level Portworx should use at installation.
+
+1. Use the steps from the [Generate the specs](/portworx-install-with-kubernetes/on-premise/other/#generate-the-specs) section to generate a Kubernetes manifest.
+
+2. Download the manifest and use the `args` section of the Portworx `DaemonSet` to specify the RAID level.
+
+  * Below is an example for specifying RAID 0:
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/advanced-installation-parameters.md]:
```suggestion
  * The following example specifies RAID 0:
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/advanced-installation-parameters.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    args:
+      ["-k", "etcd:http://70.0.0.65:2379", "-c", "kockica-11", "-d", "eth1", "-m", "eth1", "-s", "/dev/sdb", "-secret_type", "k8s", "-raid", "0",  
+      "-x", "kubernetes"]
+    ```
+
+    {{<info>}}
+**Note**: All other arguments, apart from the ones that specify the RAID level, are examples. Be sure to use values that match your environment.
+    {{</info>}}
+
+  * Below is an example for specifying RAID 10:
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/advanced-installation-parameters.md]:
```suggestion
  * The following example specifies RAID 10:
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/advanced-installation-parameters.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+    ```text
+    args:
+      ["-k", "etcd:http://70.0.0.65:2379", "-c", "kockica-11", "-d", "eth1", "-m", "eth1", "-s", "/dev/sdb", "-secret_type", "k8s", "-raid", "10",  
+      "-x", "kubernetes"]
+    ```
+
+1. Continue with the steps from the [Apply the specs](/portworx-install-with-kubernetes/on-premise/other/#apply-the-specs) section.
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/advanced-installation-parameters.md]:
```suggestion
3. Continue with the steps from the [Apply the specs](/portworx-install-with-kubernetes/on-premise/other/#apply-the-specs) section.
```

**Pull request merged**